src/frontend/apps/drive/src/hooks/useCopyToClipboard.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  addToast,
+  ToasterItem,
+} from "@/features/ui/components/toaster/Toaster";
+import { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+
+export const useClipboard = () => {
+  const { t } = useTranslation();
+
+  return useCallback(
+    (text: string, successMessage?: string, errorMessage?: string) => {
+      navigator.clipboard
+        .writeText(text)
+        .then(() => {
+          addToast(
+            <ToasterItem>
+              <span className="material-icons">check</span>
+              <span>{successMessage ?? t("Copied to clipboard")}</span>
```

Comment:
We should use dot notation for translation `toasts.clipboard.success` for instance